deployments/dockerfiles/spotbugs/mvn-entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash -eu
+#
+# Copyright 2019 Globo.com authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+#
+# This script will build every securityTest container based on all dockerfiles from huskyCI repository
```

Comment:
Ops, it looks like this script does something else, doesn't it? Would you mind adding a brief description of it? 🙃

Reply:
yeah i'll try to add a description to it